core/src/main/kotlin/com/strumenta/kolasu/traversing/Structurally.kt

```diff
@@ -96,7 +96,7 @@ fun Node.walkAncestors(): Sequence<Node> {
  */
 fun Node.walkChildren(): Sequence<Node> {
     return sequence {
-        this@walkChildren.properties.forEach { property ->
+        this@walkChildren.originalProperties.forEach { property ->
```

Comment:
maybe also `walkChildren` could get the flag `includeDerived` set to false by default